Kernel/Modules/AdminPackageManager.pm

```diff
@@ -139,6 +139,7 @@ sub Run {
             if ($Content) {
                 $MainObject->Require('Text::Diff');
                 my $Diff = Text::Diff::diff( \$File, $Content, { STYLE => 'OldStyle' } );
+                $Kernel::OM->Get('Kernel::System::Encode')->EncodeInput( \$Diff );
```

Comment:
Please prevent direct calls of the object manager. Please fetch the object first and then call the needed function:

```perl
                my $EncodeObject = $Kernel::OM->Get('Kernel::System::Encode');
                $EncodeObject->EncodeInput( \$Diff );
```
It's okay for now because I need to change stuff in the same file right now and could fix it within my changes.
I'll approve and merge, but please check this in your future pull requests.